Makefile

```diff
@@ -130,7 +131,7 @@ NM      := $(MIPS_BINUTILS_PREFIX)nm
 
 N64_EMULATOR ?= 
 
-INC := -Iinclude -Iinclude/libc -Isrc -I$(BUILD_DIR) -I.
+INC := -Iinclude -Iinclude/libc -Isrc -I$(BUILD_DIR) -I. -I$(EXTRACTED_DIR)
```

Comment:
I haven't thought deeply about it, but why this order? i.e. `.` between `BUILD_DIR` and `EXTRACTED_DIR`

Reply:
idk about BUILD_DIR and ., there may be a reason to be found in git blame but I don't remember

Why EXTRACTED at the end, afaict it currently doesn't matter and could be elsewhere. But it ties into the other discussion of EXTRACTED/assets
https://discord.com/channels/688807550715560050/688851317593997489/1208148581945708626